### PR TITLE
Feature/merge speech and person recognition

### PR DESCRIPTION
**Pull request by LoyVanBeek**

### CODE REVIEW

**Comment by kyordhel** [comment]
Reviewed.

I find the test description a little bit too long and little to do with person recognition. I will perform the following changes and push it to the same branch before proceeding with the merge:
- Shorten the test description.
- Integrate Person Recognition setup.
- Change the questions from _objects' color/shape/size/type _ to _features of the crowd, arena and objects_
  - Size of the crowd
  - Number of girls/boys/men/women in the crowd
  - Heaviest/lightest/largest/smallest object
  - Name a {color} {category} (e.g. red drink)
  - Where is the {object} (e.g. table -> kitchen)
  - And so on, but won't be specified in the rulebook
- In each phase, the robot will answer:
  - 2 questions about the crowd/operator
  - 2 questions about the arena
  - 4 questions about the object
- Check the alloted time for the test.
- State that the features will be announced during setup days (same as predefined questions).

**Comment by LoyVanBeek** [comment]
We can also use a better name. 
E.g. 

- [ ] Answering Questions (AQ)
- [ ] Questions and Answers (Q&A)
- [ ] Speech and Person Recognition (SPR)

File: tests/RoboGame_appendix.tex [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+The object-questions are built on basis of the features of the predefined objects used during the competition and their categories. Such features include color, shape, size, type, weight, category, predefined location, etc.
+
+Some example object-questions are:
+\begin{enumerate}
+    \item What's the smallest food? $\rightarrow$ \textit{The egg is the smallest in the food category}.
+    \item What's the lightest drink? $\rightarrow$ \textit{The Coke Zero, is lighter than water}.
+    \item Where can I find the tray? $\rightarrow$ \textit{The tray is in the shelf}.
+    \item Where can I find the beer? $\rightarrow$ \textit{I put it into the fridge for you, master}.
+    \item What's the color of the shampoo? $\rightarrow$ \textit{The shampoo is blue}.
+    \item What's the color of the sponge? $\rightarrow$ \textit{The sponge is yellow and has square pants}.
```
Comment (LoyVanBeek) [inline tests/RoboGame_appendix.tex]: :smile:

**Comment by hideakin21** [comment]
I reviewed.
Making story as playing a riddle game is really good to understand the task for audience.

I have to say that the speech recognition is difficult for Toyota HSR.
Its microphones do not have high performance, but reconstruction is not allowed because of SPL.
I've heard that Pepper's one is poorer.
Thus, the blind man's bluff game is especially difficult for both HSR and Pepper.

Another important thing is audibility for the audience.
During this kind of task so far, the audience could not hear what operator said. 
They could hardly understand what is going on in the task.
Robocup is not only the competition but also a kind of exhibition. 
So, we have to do our best to impress the audience.

Komei and I as LOC are now planning to setup the audio environment to improve the audibility.
We would like the operator to always talk with the microphone which is connected to the speaker for the audience.

**Comment by kyordhel** [comment]
@hideakin21  Sorry but I don't get your point. I may or may not agree with what you state but what you jus write are plain facts that lead nowhere. Could you please clarify what you suggest must be changed?

**Comment by HideakiNagano21** [comment]
@kyordhel, sorry for my lack of talk.
One of the points what I wanted to say has been stated clearly by Komei in #216. 
And reducing the number of speakers and modifying their locations are good solution to my concern. 

The other is about the audibility for the audience.
I made a figure to describe my suggestion.
2en
What do you think about this?

**Comment by kyordhel** [comment]
I thing that this has nothing to do with the _Speech and Person Recognition_ test, nor with the Toyota HSR, but rather with general rules while also afecting all leagues. I strongly advise you to start a new issue on the topic. However, first we need to
- solve the _(if needed)_ gaps and make clear requirements in the rulebook,
- make sure the local organization can provide the required external devices, and
- if any access point is needed, audio for the audience will not interfere with robot performance (specially Pepper booting).

Ohter than that, and based on my previous experience, I'm all against connecting robots to external _big_ speakers because they have this annoying costume of becoming deaf. Plus, teams always forget to return the adapter/transmitter and OC needs to chase them, but that's *my* personal opinion. As OC Chair, what needs to be done, will be done.

**Comment by HideakiNagano21** [comment]
Thanks.
I have started a new  issue #222.

**Pull request merged**